frontend/admin/src/js/components/poolCandidate/poolCandidatesOperations.graphql

```diff
@@ -106,28 +106,6 @@ fragment poolCandidateForm on PoolCandidate {
   }
   cmoIdentifier
   expiryDate
-  isWoman
```

Comment:
With the CreatePoolCandidate and UpdatePoolCandidate pages gone, some of the operations in this file are no longer used anywhere in the app. I suggest doing a search for each operation name, and if you can't find it used anywhere, delete it entirely.

Reply:
I did make an issue in the past to go through and clean graphQL files, removing redundant and unused operations in #4927 
the create/update pool candidate mutations are actually still used though by the `GeneralInformationTab` component group

getCreate and getUpdate pool candidate data removed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5055/commits/e6427000afc1de53afee09679c1137e72c65cfb4